sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/rules/XmlSonarWayProfile.java

```diff
@@ -33,19 +40,38 @@
  */
 public final class XmlSonarWayProfile extends ProfileDefinition {
 
-  private final AnnotationProfileParser annotationProfileParser;
+  private final RuleFinder ruleFinder;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this use of "RuleFinder"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)